include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
cmake/qmake/no autotools on user's side, scons/what else? :)

on Linux there are tons of build systems :)